participants/oliver.json

```
@@ -0,0 +1,54 @@
+{
+  // your real name (required by location host)
+  "realName": {
+    "givenName": "Oliver",
+    "familyName": "Tigges",
+    // if you prefer to have your family name shown first (optional)
+    "placeFamilyNameFirst": false,
+    // if you do not want to show your family name on the participant list (optional)
+    "hideFamilyNameOnWebsite": false
+  },
+  // please put in the account name of the PR creator, if you sign up somebody else
+  "githubAccountName": "otigges",
+  // company name (optional)
+  "company": "Inspired Consulting",
+  // either both days or at least one day has to be set to true.
+  // If you cannot make one of the days, please set it to false, to allow someone else take that spot!
+  "when": {
+    // June 27th, 2025
+    "friday": true,
+    // June 28th, 2025
+    "saturday": true
+  },
+  // if you are willing to take session notes and publish them to github (required)
+  "iCanTakeNotesDuringSessions": true,
+  // your current interests (JS and in general) (required)
+  "tags": [
+    "Vanilla JS",
+    "Web Development",
+    "Software Architecture",
+    "Elixir",
+    "Python"
+  ],
+  // if you only eat vegan food (optional)
+  "vegan": false,
+  // if you only eat vegan or vegetarian food (optional)
+  "vegetarian": true,
+  // what you cannot eat or drink (optional); If you don't want to put it in here, message the organizers.
+  // IMPORTANT: we cannot guarantee that food for every diet will be available,
+  //            if you have gluten free diet, please make backup plans.
+  "allergies": [],
+  // tell us a few words how JavaScript affects you (required)
+  "whatIsMyConnectionToJavascript": "First contact areound 1995, still learning.",
```

Oops, forgot to send this review comment!

Small typo?
```suggestion
  "whatIsMyConnectionToJavascript": "First contact around 1995, still learning.",
```

And wow, right after it's initial release? 🤩 What were the biggest changes for you?